net-im/slack/slack-4.4.2.ebuild

```diff
@@ -82,7 +84,8 @@ src_install() {
 
 	insinto /opt/slack
 	doins -r usr/lib/slack/.
-	fperms +x /opt/slack/slack
+	for i in $(echo -n "${QA_PREBUILT}") ; do fperms +x "$i" ; done
+	use suid && fperms u+s /opt/slack/chrome-sandbox # wrt 713094
```

Comment:
Would it do any harm if you always unconditionally just applied this?

Reply:
I don't like to have suid binaries on my system, but upstream provides them.
Personally for myself the only reason to have suid USE flag is having a possibility to unset it. On my system slack works fine without suid bit.
And the only reason why I am introducing it now is 713094. It looks like it is needed on some systems.
Another point to have it — do not make "silent bad surprise" for other users who may not like suid binaries too, especially without need.

We can remove use flag and just warn users via einfo/elog about that change for two-three ebuild bumps and remove einfo/elog after all.